cardano-config/src/Cardano/Config/Orphanage.hs

```diff
@@ -18,10 +20,22 @@ import qualified Data.Text as T
 
 import           Cardano.BM.Data.Tracer (TracingVerbosity(..))
 import qualified Cardano.Chain.Update as Update
+import           Cardano.Crypto.DSIGN.Ed448 (Ed448DSIGN)
+import           Cardano.Crypto.KES (SignKeyKES, SimpleKES, VerKeyKES)
 import qualified Ouroboros.Consensus.BlockchainTime as Consensus
 import           Ouroboros.Consensus.NodeId (NodeId(..), CoreNodeId (..))
+import           Ouroboros.Consensus.Shelley.Protocol.Crypto (TPraosStandardCrypto)
+import           Shelley.Spec.Ledger.Keys (VKeyES (..))
 
 
+deriving instance Generic (VKeyES TPraosStandardCrypto)
+deriving instance NFData (VKeyES TPraosStandardCrypto)
+
+deriving instance Generic Ed448DSIGN
+deriving instance NFData (SignKeyKES (SimpleKES Ed448DSIGN))
+
+deriving instance NFData (VerKeyKES (SimpleKES Ed448DSIGN))
```

Comment:
Might be nicer if, instead of directly specifying the `DSIGNAlgorithm` `Ed448DSIGN`, we do this:

```suggestion
deriving instance NFData (VerKeyKES (KES TPraosStandardCrypto))
```